[2024-02-29 13:23:22,734] [INFO] [server_logger.py:29] Logger initialized
[2024-02-29 13:23:22,777] [INFO] [login.py:23] Connection Successful
[2024-02-29 13:23:22,781] [INFO] [register.py:18] Connection Successful
[2024-02-29 13:23:22,785] [INFO] [survey.py:14] Connection Successful
[2024-02-29 13:24:41,588] [INFO] [login.py:38] Successful login for user: aaa@aaa.aaa
[2024-02-29 13:24:41,589] [INFO] [login_controller.py:17] Login successful
[2024-02-29 13:25:29,840] [ERROR] [register.py:49] Error in register_model: 'username'
[2024-02-29 13:25:29,841] [INFO] [register_controller.py:24] User registration successful
